test/cli/simple-package/simple-package.out

```diff
@@ -2,15 +2,15 @@ foo/foo.rb:11: Unable to resolve constant `Bard` https://srb.help/5002
     11 |  Project::Bar::Bard
           ^^^^^^^^^^^^^^^^^^
   Autocorrect: Use `-a` to autocorrect
-    foo/foo.rb:11: Replace with `Project::Bar::Bar`
+    foo/foo.rb:11: Replace with `Project::Bar`
     11 |  Project::Bar::Bard
           ^^^^^^^^^^^^^^^^^^
   Autocorrect: Use `-a` to autocorrect
-    foo/foo.rb:11: Replace with `Project::Bar`
+    foo/foo.rb:11: Replace with `Project::Bar::Bar`
     11 |  Project::Bar::Bard
           ^^^^^^^^^^^^^^^^^^
-    foo/__package.rb: Did you mean: `Project::Bar::Bar`?
     foo/__package.rb:5: Did you mean: `Project::Bar`?
      5 |  import Project::Bar
                  ^^^^^^^^^^^^
+    foo/__package.rb: Did you mean: `Project::Bar::Bar`?
```

Comment:
These changes worry me.  Are we sorting on the raw id somewhere, and reordering the bits has changed things?  Is something reaching into the innards of `NameRef`/`SymbolRef` and didn't get updated for this change?  Or something else?

Reply:
Yes, `findMemberFuzzyMatchConstant` sorts on rawId:

https://github.com/sorbet/sorbet/blob/03f5d54c/core/Symbols.cc#L569

That's what is used for `Did you mean:` suggestions.

Reply:
(The sorting is done for determinism reasons, not for semantic reasons.)

Reply:
Works for me, thanks for clearing this up!